explainer.md

```diff
@@ -550,27 +565,27 @@ should then be.
 There are 3 main possible approaches to consider, if the integrity of the PEPC
 click is not assured:
 
-- The click triggers the legacy permission flow (as if it was triggered by the
-  equivalent JS API). This approach is worth considering if the failing check or
-  mitigation is not something self-correcting (e.g. styling issue or the PEPC
-  being covered).
-- The click does nothing. This approach is worth considering if the failing
-  check or mitigation will self-correct itself (e.g. if the PEPC has moved
-  recently there will be a short cooldown before the PEPC integrity is
-  restored).
-- The PEPC could be corrected by the user agent itself in order to preserve its
-  integrity. For example, if the style specified by the site sets the PEPC font
-  to be too small to read, this can be corrected by the user agent by forcing a
-  minimum font size on the PEPC. This should be considered primarily in the case
-  of CSS, which the user agent can override as it sees fit. **Open question:**
-  should there be a way for sites to specify whether they want to allow the user
-  agent to override the style? Some site authors might be happier triggering the
-  legacy prompt request flow, rather than have the PEPC style be changed whereas
-  others might prioritize the benefits of a PEPC permission flow over making
-  sure the style is exactly as desired. User agents need to weigh in the
-  additional flexibility afforded to site authors against the potential user
-  confusion of seeing the PEPC permission prompt vs the regular permission
-  prompt.
+-   The click triggers the legacy permission flow (as if it was triggered by the
```

Comment:
This is also the bit we (WebKit) are finding troubling, tbh. We went down this path previously with `permission.request()` and ultimately rejected the idea because it was duplicative of what's already implemented via JS.

The approach we prefer is to look at if we can fix things at the API level: 

 * https://github.com/w3c/geolocation-api/issues/48
 * https://github.com/whatwg/notifications/issues/108

Reply:
Extracted into https://github.com/WICG/PEPC/issues/8